### ISSUE TITLE
Bitcoin Miner Detected???

### ISSUE THREAD

**Issue by opplayzminecraf** [TheDuckCow/MCprep#195]
I don't know why but whenever I download MCprep with Norton360, it says it has a bitcoin miner. DuckCow might need to review why its detecting a bitcoin miner if it isn't mining bitcoin...

**Comment by TheDuckCow** [comment]
Thanks for raising the issue in case there's an issue. That being said, I'm doubtful there is - and to be quite explicit, I have in no way inserted a bitcoin miner (or any other malicious intent) as a part of MCprep, and see there have bene no injected code commits. 

I am not sure how this could be happening unless 1) your own network is somehow compromised or 2) GitHub itself is compromised (there would be some news and alerts going on about this if so).

The actual files for MCprep are saved to a .zip file, hosted by github. The download page on TheDuckCow.com just points to these files. I've just inspected the download itself out of an abundance of caution, and found no issues.
![mcprep hosting](https://user-images.githubusercontent.com/2958461/113489043-c61dad80-948f-11eb-86ab-b76b0a35a339.png)


Long story short, I think it's a false alarm by Norton360, but please do let me know if any new evidence comes up to the contrary!
